src/bitmessagemain.py

```diff
@@ -502,10 +384,85 @@ def main():
     mainprogram = Main()
     mainprogram.start()
 
+# See "workprover/Readme.md"
+
+import os
+import sys
+
+app_dir = os.path.dirname(os.path.abspath(__file__))
+os.chdir(app_dir)
+sys.path.insert(0, app_dir)
+
+import pyelliptic.openssl
+import workprover.dumbsolver
+
+workprover.dumbsolver.libcrypto = pyelliptic.openssl.OpenSSL._lib
 
 if __name__ == "__main__":
-    main()
+    import multiprocessing
```

Comment:
Please place all the imports in the starting the file...

Reply:
These imports must be conditional.